app/jobs/channelJobs.js

```diff
@@ -638,7 +638,13 @@ var job = {
                 },
                 'Open': {
                     getEndPoint: function () {
-                        return jiraSettings.record.organization + '/rest/api/3/search?jql=project in (' + jiraSettings.record.key + ') AND status!=Done AND id in (' + jiraId.toString() + ') &startAt=' + request.page * request.pageSize + '&maxResults=' + request.pageSize + '';
+                        if (request.page === "All") {
```

Comment:
what does All means here? is it fetching all jira tickets...just restrict to top 10 records

Reply:
yes, is it fetching all jira tickets
while sending open jira notification need to fetch all open jira

Reply:
can u fetch top 10 tickets by default. it;s not good practice to fetch all record in single call